docker/Dockerfile-multi-arch

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.18.3 as build
+WORKDIR /workspace
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct \
+    BINARY=postgres-operator
+COPY . .
+RUN GOOS=linux GOARCH=$(go env GOARCH) CGO_ENABLED=0 go build -o build/linux/${BINARY} ./cmd/main.go
+
+FROM registry.opensource.zalan.do/library/alpine-3.15:latest
```

Comment:
it is actually an amd-only image. You better use the official one
But what is the reason for creating a separate Dockerfile for a multi-arch build?

Reply:
1. `registry.opensource.zalan.do/library/alpine-3.15:latest` I'm not sure if it's the same as `docker.io/library/alpine-3.15:latest`
2. Dockerfile-multi-arch was created because I didn't want to break existing code.

Reply:
it is not the same but we can make image configurable via build arg and use different base images for different purposes

Reply:
About using different base images I don't quite understand what you want to do, can you give more hints? 
@hughcapet

Reply:
@hughcapet Hi, PTAL.